src/liballoc/tests/vec_deque.rs

```diff
@@ -928,6 +928,60 @@ fn test_append() {
     assert_eq!(a.iter().cloned().collect::<Vec<_>>(), []);
 }
 
+#[test]
+fn test_append_advanced() {
+    fn check(
```

Comment:
Check *what*? Tests are a form of documentation. All tests "check" something, that's rather their purpose.